lib/symbol/read.c

```diff
@@ -357,8 +357,8 @@ SYMBOL *S_read(const char *sname)
 		sscanf(data, "%lf %lf %lf %lf %lf %c", &x, &y, &rad, &ang1,
 		       &ang2, &clock);
 	    if (ret < 5) {
-		sprintf(buf, "Incorrect arc definition: '%s'", buf);
-		return (err(fp, symb, buf));
+		sprintf(buf2, "Incorrect arc definition: '%s'", buf); //
```

Comment:
```suggestion
		sprintf(buf2, "Incorrect arc definition: '%s'", buf);
```
Please no `//` comments in C code.

Reply:
Any idea how to avoid the below warning about "Uncontrolled format string" ?